gdal/src/main/scala/geotrellis/contrib/vlm/gdal/GDALWarpOptions.scala

```diff
@@ -243,4 +244,7 @@ case class GDALWarpOptions(
       dstFile orElse dstFile
     )
   }
+
+  def isDefault: Boolean = this == GDALWarpOptions()
```

Comment:
`GDALWarpOptions.Empty` would be a nice default for this can be matched against.

Reply:
`GDALWarpOptions.EMPTY` and `def isEmpty` would be good. I don't like the word `Default` at all.